### PR TITLE
Brand new scripts: Better, Faster, Stronger

### PR DESCRIPTION
**Pull request by bourretp**

### CODE REVIEW

**Comment by bourretp** [comment]
As I cannot fully test the new scripts without performing a real release, a careful review would be greatly appreciated).

File: scripts/common.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+################################################################################
+# Roboconf repository locations & directory layouts.
+# /!\ DO *NOT* EDIT unless you know perfectly what you're doing!!!
+################################################################################
+readonly ROBOCONF_ECLIPSE="roboconf-eclipse"
+readonly ROBOCONF_EXAMPLES="roboconf-examples"
+readonly ROBOCONF_LANGUAGE="language-roboconf"
```
Comment (vincent-zurczak) [inline scripts/common.sh:39]: We don't need to release it.
Same thing for the maven archetype and the parent.

File: scripts/eclipse.sh [hunk 2]
```diff
@@ -1,40 +1,110 @@
...
+# Import the commons.
+source common.sh
+
+
+
+echo
+echo "Checking out the eclipse plugin..."
+echo
+
+DIR="$(localStagingDirectory ${ROBOCONF_ECLIPSE})"
```
Comment (vincent-zurczak) [inline scripts/eclipse.sh:37]: I have never seen and used this notation...
Maybe a lack from me.

**Comment by vincent-zurczak** [comment]
Except this notation that I have never seen, it looks good to me. :sweat_smile: 
Good job! :+1:  I will wait for your feedback and merge it right after.

**Comment by bourretp** [comment]
`$(…)` stands for ``…``. POSIX defines both as equivalent. However the first syntax allows easier (and more readable) nesting as in `$(cat /proc/$(pidof java)/cmdline)`

More generally, I've tried to stick to the [Google shell style guide](https://google-styleguide.googlecode.com/svn/trunk/shell.xml).

**Pull request merged**